client/src/components/VtkViewer.vue

```diff
@@ -241,7 +391,7 @@ export default {
   bottom: 0;
   left: 0;
   right: 0;
-  background: linear-gradient(#3a3a3a, #1d1d1d);
+  background: gray;
```

Comment:
Is this change intentional or leftover from debugging?

Reply:
good catch, I meant to change that to solid black so the image looks seamless against the backdrop. I think it looks cleaner than the linear gradient. What do you think?